.github/workflows/federation-v2.workflow.yaml

```diff
@@ -1,53 +0,0 @@
-name: federation-v2
```

Comment:
Why do we remove Fed v2?

Reply:
Adding it back, it was purely to get CI to do less, while rapidly benchmarking to reduce runner minutes consumption.